_Console/BuildDevSave/readme.md

```diff
@@ -1,3 +1,11 @@
 This was originally intended to be extended into a general-purpose Lua script reader to have access to QueryCiv3, but the immediate need was to export select Civ3 map data to JSON for temp use in C7-Game/Prototype. After I got Lua working and printing out the info I realized it's simpler just to skip Lua. This is now both acting as a tester for the import Civ3 save function of C7GameData and to set the default/hard-coded map for C7Game.
 
 The script file reference, save file path, and civ3 path are hard-coded for now, and it's currently unclear how long we'll need this, but it might be nice to parameterize this.
+
+#### Build & Use
+
+- Either registry or CIV3_HOME must be pointing to the Civ3 home folder
```

Comment:
This isn't actually accurate, for BuildDevSave.  It still looks at /Users/jim/civ3 .

I'm okay with leaving it at the --assume-unchanged workaround and changing it locally if it's a hassle, but since the readme now says it'll use CIV3_HOME, I wanted to mention it.

Reply:
Oh, oops. I'll change that. Hate to have flat-out wrong documentation, especially from the same commit.